doc/Language/signatures.rakudoc

```diff
@@ -1035,9 +1036,6 @@ c (Int);        #Undefined
 c 42;           #Defined
 =end code
```

Comment:
I think that you are correct to use the `===` value identity operator (obviously I was having a bad day when I wrote this!)

As I re-read this section with fresh eyes, I realise that we should be clearer that this test is True if no argument is passed (ie you identity matching the default in the absence of a passed argument).